test/integration/cases/07-sni-route/verify.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ex
+IFS=: read sut_hostname sut_port <<< "$SUT_HTTPS_HOST"
+
+[ "$(curl -k -sw '%{http_code}' -o /dev/null https://example.com:27443/foo --resolve example.com:$sut_port:$sut_hostname)" == 200 ]
```

Comment:
did you mean `$sut_port`?

Reply:
The curl syntax for `--resolve` is weird (port inclusion is for simulating SRV maybe?), but it is what it is:

```
$ man curl | grep "\-\-resolve <"
       --resolve <host:port:addr[,addr]...>
```

Reply:
Sorry about an ambiguous comment.

I meant that the test contains a magic number `27443`. Did you intend to use `$sut_port` instead?

Reply:
Ah, damnit, I cannot read. They now both use the port variable.